modules/cli/src/main/scala/scala/cli/commands/ShebangOptions.scala

```diff
@@ -3,7 +3,9 @@ package scala.cli.commands
 import caseapp._
 
 @HelpMessage(
-  """|This command is an equivalent of `run`, but it changes the way how
+  """|Access to command line arguments in shebang scripts
```

Comment:
I'm not sure about that exact wording… Maybe:
```suggestion
  """|Like 'run', but more handy from shebang scripts
```

Reply:
It sounds better.